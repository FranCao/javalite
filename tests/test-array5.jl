void repeatStr(string[] arr, int len) {
    int i;
    for (i = 0; i < len; i = i + 1) {
        arr[i] = concat(arr[i], arr[i]);
    }
}

int main() {
    string[] strArr;
    strArr = ["hi", "hello", "hey"];
    print(strArr[0]);
    print(strArr[1]);
    print(strArr[2]);
    repeatStr(strArr, 3);
    print(strArr[0]);
    print(strArr[1]);
    print(strArr[2]);
    return 0;
}