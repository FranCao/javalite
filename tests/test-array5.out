hi
hello
hey
hihi
hellohello
heyhey